packages/fhir-router/src/graphql.ts

```diff
@@ -65,7 +69,7 @@ import {
 import { FhirRequest, FhirResponse, FhirRouter } from './fhirrouter';
 import { FhirRepository } from './repo';
 
-const typeCache: Record<string, GraphQLOutputType | undefined> = {
+const typeCache: Record<string, GraphQLObjectType | GraphQLInputType | undefined> = {
```

Comment:
You might be able to use `GraphQLScalarType` here rather than `GraphQLObjectType | GraphQLInputType`